test/compilable/iasm64.d

```diff
@@ -0,0 +1,24 @@
+// PERMUTE_ARGS:
+
+version (D_InlineAsm_X86_64)
+{
+    void b18553()
+    {
+        asm
+        {
+            mov RAX, CR0;
```

Comment:
Please add to test/runnable/iasm64.d instead, as those include checks that the correct opcodes are emitted. Just seeing if it compiles is insufficient. You can see these tests for these instructions in 32 bit mode in test/runnable/iasm.d.

Reply:
Thanks! You can delete this file now :-)